connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -339,6 +352,22 @@ public static synchronized void sdkInitialize(Context context) {
                         RestHelper.getConnectApi(getConnectApiUrl().toString()),
                         profile.getClientId(),
                         profile.getRedirectUri()));
+
+        initializeCommonComponents();
+    }
+
+    public static String getMccMnc() {
+        TelephonyManager tel
+                = (TelephonyManager)getContext().getSystemService(Context.TELEPHONY_SERVICE);
+        return tel.getNetworkOperator();
+    }
+
+    public static String getMccMncLoginHint() {
```

Comment:
Could you pls clarify? MCC MNC are quite standard telco words.
I moved the getMccMncLoginHint though to ConnectLoginButton as it seems this method has much more knowledge for the OAuth query param names.